app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,41 @@
+class TodosController < ApplicationController
+  before_action :set_todo, only: [:show, :update, :destroy]
+
+  def index
+    @todos = current_user.todos
+    json_response(@todos)
+  end
+
+  def show
+    json_response(@todo)
+  end
+
+  def update
+    @todo.update(todo_params)
+    json_response(@todo, :ok)
+  end
+
+  def destroy
+    Todo.delete(@todo)
```

Comment:
클래스 메소드로 삭제가 가능한구요! 보통 다른 crud에서 구현해주신 것처럼 모델 객체를 직접 참조해서 delete 하는게 일반적입니다! 

그리고 `delete` vs `destroy` 에 대해 한번 추가적으로 스터디해보시길 추천드립니다.

```suggestion
    @todo.delete
```

Reply:
제안 주신대로 Todo.delete(@todo) 보다는 해당 ActiveRecord 인스턴스를 직접 delete를 호출하는 것이 일반적이라고 생각되네요! 이 부분은 수정하도록 하겠습니다.

Reply:
제안주신대로 ActiveRecord의 delete와 destroy의 차이점에 대해서 검토해봤습니다.
결론먼저 말씀드리면 destroy 메서드를 쓰는게 더 적합할 것 같네요.
그럼 최종 수정 코드는 `@todo.destroy`가 되겠네요.

`delete`와 `destroy`의 가장 큰 차이는 호출시 콜백과 연관관계를 처리 여부라고 생각합니다.
`delete`의 경우 레코드 삭제 전후 처리를 위한 `before_destory`나 `after_destroy` 메서드를 호출하지 않습니다.
또한, `dependant: :destroy`와 같은 연관관계 처리도 무시됩니다.

반면, `destory`의 경우 삭제 전후 처리를 위한 콜백 메서드와 연관관계 처리를 포함해서 레코드를 삭제합니다.
그렇기때문에 `destory`가 ActiveRecord 표준 삭제 메서드로 불리우고있고요.

현재 프로젝트에서는 `todo`가 `items`와 연관관계가 있기 때문에 `todo` 삭제시 `items`도 함께 삭제되어야합니다.
그래서 더더욱이 `destory` 메서드를 사용하는게 적합할  것 같네요